### PR TITLE
CI: Add building documentation

### PR DESCRIPTION
**Pull request by azhogov**

### CODE REVIEW

File: .github/workflows/build_doc.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          sudo make install
+
+      - name: CMake doc
+        run: |
+          mkdir openvino_doc
+          cd openvino_doc
+          cmake -DENABLE_DOCS=ON ..
+
+      - name: Build doc
+        run: cmake --build . --target openvino_docs || exit 0
```
Comment (ilya-lavrenov) [inline .github/workflows/build_doc.yml:38]: looks like a hack

File: .github/workflows/build_doc.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - name: Clone OpenVINO
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          lfs: true
+
+      - name: Install dependencies
+        run: |
+          sudo apt --assume-yes install libusb-1.0-0-dev graphviz texlive
+          python3 -m pip install -r ./inference-engine/ie_bridges/python/requirements.txt
```
Comment (ilya-lavrenov) [inline .github/workflows/build_doc.yml:17]: why do we need `ie_bridges` requirements?

File: .github/workflows/build_doc.yml (hunk 1 above)
Comment (azhogov) [reply .github/workflows/build_doc.yml:38]: It is temporary hack until fix in the doc

File: .github/workflows/build_doc.yml (hunk 2 above)
Comment (azhogov) [reply .github/workflows/build_doc.yml:17]: It was needed for cmake, probably not needed now

**Review by ntyukaev — APPROVED** [review]

File: .github/workflows/build_doc.yml (hunk 1 above)
Comment (ilya-lavrenov) [reply .github/workflows/build_doc.yml:38]: if the documentation build is broken, we cannot catch errors. What is benefits of adding documentation build, but ignore the result?

File: .github/workflows/build_doc.yml (hunk 2 above)
Comment (ilya-lavrenov) [reply .github/workflows/build_doc.yml:17]: can we remove?

File: .github/workflows/build_doc.yml (hunk 1 above)
Comment (azhogov) [reply .github/workflows/build_doc.yml:38]: It was needed for checking whole pipeline. It is fixed now, see master

Comment (azhogov) [reply .github/workflows/build_doc.yml:38]: removed

File: .github/workflows/build_doc.yml (hunk 2 above)
Comment (azhogov) [reply .github/workflows/build_doc.yml:17]: removed

**Pull request merged**